packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -3,54 +3,58 @@ import envVariables from '../../env-variables';
 import { translateFromPage } from '../utils';
 import type { EditorPreviewOptions } from './types';
 
-const panel = 'div.interface-interface-skeleton__header';
+const postEditorHeader = 'div.interface-interface-skeleton__header';
+const siteEditorHeader = 'div.edit-site-layout__header';
```

Comment:
**suggestion:non-blocking** - FYI, I was able to reliably select the toolbar element using `playwright.getByRole("region")` in the console by launching a debug instance. This will be i18n-safe as I tested this in both Japanese and English UI editors. 

If methods within the `EditorToolbarComponent` is prepended with `this.editor.getByRole("region")` then that should eliminate the need to pass in the `header` to each selector string.

```
eg.
const locator = this.editor.getByRole('region').first().locator('')
```

Up to you if you want to use this suggestion, I won't block merge on this.

Reply:
I think these changes are no longer needed because of https://github.com/Automattic/wp-calypso/pull/72112, so I'll see if I can revert them as soon as we're able to switches the site's theme back to a block theme.